tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -285,16 +318,51 @@ public static IList<TestEndpoints> GetAllTestEndpoints()
                     new ("basketservice", "Application started"),
                     new ("postgres", "database system is ready to accept connections"),
                 ]),
+            // Test endpoints disabled for AppHostRunsCleanlyForAppsWithNoOtherTests as they have specific tests
+            new TestEndpoints("WithDockerfile.AppHost", []),
+            new TestEndpoints("KafkaBasic.AppHost", []),
+            new TestEndpoints("AzureFunctionsEndToEnd.AppHost", [])
         ];
 
         return candidates;
     }
 
+    private static async Task TestEventHubsAppHost(DistributedApplication app, string appHostPath, ITestOutputHelper testOutput)
+    {
+        using var client = CreateHttpClientWithResilience(app, "api");
+
+        var consumerMessage = "Hello, from /test sent via producerClient";
+        var consumerMessageLoggedTask =
+                app.WaitForTextAsync(log => log.Contains(consumerMessage), resourceName: "consumer")
+                    .WaitAsync(TimeSpan.FromMinutes(2))
+                    .ConfigureAwait(false);
```

Comment:
Why is this declared up here? Why not just inline it in the `await` inside the `try catch` below?

Reply:
I'm trying to avoid missing this message.

Reply:
Doesn't it read through the whole log from the start? If it already happened, I think it would still get seen.

Reply:
You are correct, it does play the whole log from the start! Maybe it should be renamed to like `FindOrWaitForTextAsync`.